internal/plugins/helm/v1/scaffolds/api.go

```diff
@@ -18,90 +18,88 @@ limitations under the License.
 package scaffolds
 
 import (
-	"errors"
 	"fmt"
 	"os"
-	"path/filepath"
 
+	"github.com/spf13/afero"
 	"helm.sh/helm/v3/pkg/chart"
-	"helm.sh/helm/v3/pkg/chartutil"
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
-	"sigs.k8s.io/kubebuilder/v3/pkg/model"
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
 	"sigs.k8s.io/kubebuilder/v3/pkg/model/resource"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins"
 
-	"github.com/operator-framework/operator-sdk/internal/kubebuilder/cmdutil"
-	"github.com/operator-framework/operator-sdk/internal/kubebuilder/machinery"
-	internalchartutil "github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/chartutil"
+	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/chartutil"
 	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/internal/templates"
 	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/internal/templates/config/crd"
 	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/internal/templates/config/rbac"
 	"github.com/operator-framework/operator-sdk/internal/plugins/helm/v1/scaffolds/internal/templates/config/samples"
 )
 
-var _ cmdutil.Scaffolder = &apiScaffolder{}
+var _ plugins.Scaffolder = &apiScaffolder{}
 
 // apiScaffolder contains configuration for generating scaffolding for Go type
 // representing the API and controller that implements the behavior for the API.
 type apiScaffolder struct {
+	fs afero.Fs
+
 	config   config.Config
-	resource *resource.Resource
-	chrt     *chart.Chart
+	resource resource.Resource
+	chart    *chart.Chart
 }
 
-// NewAPIScaffolder returns a new Scaffolder for API/controller creation operations
-func NewAPIScaffolder(config config.Config, res *resource.Resource, chrt *chart.Chart) cmdutil.Scaffolder {
+// NewAPIScaffolder returns a new plugins.Scaffolder for API/controller creation operations
+func NewAPIScaffolder(cfg config.Config, res resource.Resource, chrt *chart.Chart) plugins.Scaffolder {
 	return &apiScaffolder{
-		config:   config,
+		config:   cfg,
 		resource: res,
-		chrt:     chrt,
+		chart:    chrt,
```

Comment:
nit: came back to chrt which is how it is defined in helm code implementation

Reply:
My thoughts: `chart` is obviously better than `chrt` as a variable name, but `chart` is the import, so use `chart` when able but not as a variable name because it would conflict with the import alias.

Reply:
I thought that ib the past too. Then, @joelanford who is the helm operator father suggested reverting because in the helm pkg is chrt